authenticator/server/config/config.go

```diff
@@ -136,10 +142,23 @@ func setConfigEnvVars() error {
 		return err
 	}
 
+	if err := viper.BindEnv("ConfigFilePath", "APPROZIUM_CONFIG_FILE_PATH"); err != nil {
+		return err
+	}
+
 	if err := viper.BindEnv("VaultTokenPath", "APPROZIUM_VAULT_TOKEN_PATH"); err != nil {
 		return err
 	}
-	if err := viper.BindEnv("ConfigFilePath", "APPROZIUM_CONFIG_FILE_PATH"); err != nil {
+
+	if err := viper.BindEnv("VaultToken", "VAULT_TOKEN"); err != nil {
+		return err
+	}
+
+	if err := viper.BindEnv("VaultAddr", "VAULT_ADDR"); err != nil {
+		return err
+	}
+
+	if err := viper.BindEnv("AwsRegion", "AWS_REGION"); err != nil {
```

Comment:
Should we also support `AWS_DEFAULT_REGION` here? Or support it instead? Looking at env vars [here](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html).

Reply:
Good catch! Upon further googling, it looks like there was an [issue](https://github.com/aws/aws-sdk-go/issues/2103) about this for the golang aws sdk, and they said they wanted to stick to `AWS_REGION`. Since we read the env variable manually and put it in the SDK, we can change our behaviour. However, for simplicity, I think sticking to one of the env variables is better than supporting two.